lib/ddr/models/solr_document.rb

```diff
@@ -248,6 +252,24 @@ def rights
       self["rights_tesim"]
     end
 
+    def original_filepath
+      if self[Ddr::Index::Fields::ORIGINAL_FILEPATH]
+        self[Ddr::Index::Fields::ORIGINAL_FILEPATH]
+      elsif self[Ddr::Index::Fields::ACTIVE_FEDORA_MODEL] == 'Item'
+        if childrn = children
```

Comment:
just so I don't call the "children" method multiple times, re-executing the query each time